broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -212,10 +212,10 @@ public static BrokerController createBrokerController(String[] args) {
             MixAll.printObjectProperties(log, messageStoreConfig);
 
             final BrokerController controller = new BrokerController(
-                brokerConfig,
-                nettyServerConfig,
-                nettyClientConfig,
-                messageStoreConfig);
+                    brokerConfig,
```

Comment:
It seems that only the indention is changed in this file. 
IMHO, It's better keep as it is, because it will change the log history, and it will be confusing if someone want to check the implementation details through the git log.

Reply:
ok ,i reduction this file